lib/model/sdp/connection.py

```diff
@@ -209,6 +209,12 @@ def on_disconnect(self, by=None):
         if self._params[PLUGIN_ATTR_CB_ON_DISCONNECT]:
             self._params[PLUGIN_ATTR_CB_ON_DISCONNECT](by)
 
+    def check_command(self, command, value):
```

Comment:
Braucht es das schon in der connection, oder würde das im protocol reichen? Das resend-Protokoll sollte das - eigentlich - "intern" handeln, dann müsste die protocol-API reichen...?

Reply:
braucht es glaub nicht, ne